application/controllers/UploaderController.php

```diff
@@ -250,7 +327,7 @@ public function run($actionID)
                                 "name"          => rawurlencode(basename((string) $filename)),
                                 "ext"           => $cleanExt,
                                 "filename"      => $randfilename,
-                                "msg"           => gT("The file has been successfully uploaded.")
+                                "msg"           => $event->get('msg', gT("The file has been successfully uploaded."))
```

Comment:
I wouldn´t use the $event variable here. I would have a local variable to handle the message output from the event, consistent with all others output variables from the event.

Reply:
I use it like that because the second parameters is the default (if not set),

Need to do :

`$msg = $event->get('msg');`

`"msg"           => !empty($msg) ? $msg : gT("The file has been successfully uploaded.");`

But OK : let's go.